pkg/kubernetes-utils.go

```
@@ -196,6 +198,10 @@ func WaitForDeploymentToBeRolledOut(useInClusterConfig bool, deploymentName stri
 		if err != nil {
 			return err
 		}
+		retries = retries + 1
+		if retries >= maxWaitForDeploymentRetries {
+			fmt.Errorf("Timed out waiting for deployment %q", deployment.Name)
```

should we return the error?

---

indeed. just updated it